frontend/app/src/store/history/state.ts

```diff
@@ -18,9 +18,12 @@ export const defaultState = (): HistoryState => ({
   ignored: {},
   associatedLocations: [],
   trades: defaultHistoricState<TradeEntry>(),
+  tradeLocationsToPurge: [],
   assetMovements: defaultHistoricState<AssetMovementEntry>(),
+  assetMovementLocationsToPurge: [],
   transactions: defaultHistoricState<EthTransactionEntry>(),
-  ledgerActions: defaultHistoricState<LedgerActionEntry>()
+  ledgerActions: defaultHistoricState<LedgerActionEntry>(),
+  ledgerActionLocationsToPurge: []
```

Comment:
In general, what I had in mind as an approach to this didn't require additions to the state, do you think we can take that approach here?